### PR TITLE
ESP32: Added ota-requestor for lighting app and updated readme.

### PR DESCRIPTION
**Pull request by jadhavrohit924**
Removed applyupdate command from ota-requestor-app as
ota-announce automatically apply the updates once transfer completes.
Minor readme changes.

#### Problem
What is being fixed?  Examples:
* ota-requestor was not there for lighting app.
* applyupdate command was not needed as ota-announce automatically apply the updates once transfer completes.
* Minor readme changes.

#### Change overview
Added ota-requestor for lighting app.
Removed applyupdate command from ota-requestor-app/esp32.
Some minor fixed in readme

#### Testing
How was this tested? (at least one bullet point required)
* manually tested ota-requester-app/esp32 using linux ota-provider.
* manually tested  Lighting app ota requestor using linux ota-provider.

### CODE REVIEW

File: examples/ota-requestor-app/esp32/README.md [hunk 1]
```diff
@@ -40,20 +40,13 @@ After commissioning is successful, announce OTA provider's presence using
 chip-tool. On receiving this command OTA requestor will query for OTA image.
 
 ```
-./out/debug/chip-tool otasoftwareupdaterequestor announce-ota-provider 12345 0 0 12346 0
+./out/debug/chip-tool otasoftwareupdaterequestor announce-ota-provider 12345 0 0 0 12346 0
```
Comment (dhrishi) [inline examples/ota-requestor-app/esp32/README.md:43]: What parameter has been added in this command?

Comment (jadhavrohit924) [reply examples/ota-requestor-app/esp32/README.md:43]: endpoint

**Review by woody-apple — APPROVED** [review]

File: examples/bridge-app/esp32/README.md [hunk 2]
```diff
@@ -242,6 +242,6 @@ Parameters:
 To use the Client to send Matter commands, run the built executable and pass it
 the target cluster name, the target command name as well as an endpoint id.
 
-    $ .out/debug/chip-tool onoff on 12344321 1
+    $ ./out/debug/chip-tool onoff on 12344321 1
```
Comment (dhrishi) [inline examples/bridge-app/esp32/README.md:245]: This README will be updated soon. For now, please replace the endpoint 1 to 2 in the onoff command as the light isn't bridged on endpoint 1

**Review by bzbarsky-apple — APPROVED** [review]

File: examples/lighting-app/esp32/README.md [hunk 3]
```diff
@@ -122,3 +123,39 @@ make sure the IDF_PATH has been exported(See the manual setup steps above).
     control the color attributes:
 
         $ ./out/debug/chip-tool colorcontrol move-to-hue-and-saturation 240 100 0 0 0 12345 1
+
+# Steps to Try Lighting app OTA Requestor
```
[the remark this answers is not in the document]
Comment (jadhavrohit924) [reply examples/lighting-app/esp32/README.md:127]: Done

File: examples/lighting-app/esp32/README.md [hunk 4]
```diff
@@ -122,3 +123,39 @@ make sure the IDF_PATH has been exported(See the manual setup steps above).
...
+-   Build the Linux OTA Provider
+
+```
+scripts/examples/gn_build_example.sh examples/ota-provider-app/linux out/debug chip_config_network_layer_ble=false
+```
+
+-   Run the Linux OTA Provider with OTA image.
+
+```
+./out/debug/chip-ota-provider-app -f hello-world.bin
```
[the remark this answers is not in the document]
Comment (jadhavrohit924) [reply examples/lighting-app/esp32/README.md:140]: Done

File: examples/bridge-app/esp32/README.md (hunk 2 above)
Comment (jadhavrohit924) [reply examples/bridge-app/esp32/README.md]: Done

**Review by dhrishi — APPROVED** [review]

**Pull request merged**